src/app/search-page/search-page.component.ts

```diff
@@ -101,6 +101,13 @@ export class SearchPageComponent implements OnInit {
         this.patientService.getPatientData().subscribe(
           (next) => {
             if (next.total) this.setLoadingProgress(next.loaded, next.total);
+            // Add the entries of the patient resource to the bundle resources array.
+            this.bundleResources.push(
+              ...(next.entries.filter((record) => {
+                // Check to make sure it's a bundle entry
+                return 'fullUrl' in record && 'resource' in record;
+              }) as BundleEntry[])
+            );
```

Comment:
Doing this should double-add entries as the final "load" event is a "complete" event that includes all the entries. Instead it should just be

```javascript
if (next.type === 'complete') {
  this.bundleResources = next.entries;
}
```

(The filter is unnecessary because `patient.service.ts` does it.)

Reply:
Good catch, fixed it!